src/PeraWalletConnect.ts

```diff
@@ -186,6 +187,24 @@ class PeraWalletConnect {
       throw new Error("PeraWalletConnect was not initialized correctly.");
     }
 
+    if (isMobile()) {
+      let windowObj;
+
+      try {
+        // This is to automatically open the wallet app when trying to sign with it.
+        windowObj = window.open(PERA_WALLET_APP_DEEP_LINK, "_blank");
```

Comment:
Let's clean up inside of the `PeraWalletRedirectModal` component since we've moved the `window.open` stuff here.